docs/_docs/Starters Academy (LDSSA)/01-Starters-Academy-(Course).md

```diff
@@ -38,52 +44,72 @@ we will go over the following topics:
 
 ### Bootcamp Details
 
-The _Bootcamp_ is a 2-day event, on July 24th (Saturday) and 25th (Sunday). The first hackathon is three weeks after the bootcamp, and will be held on August 15th (Sunday).
+The _Bootcamp_ is a 2-day event, on July 24th (Saturday) and 25th (Sunday). The first hackathon is 
+three weeks after the bootcamp, and will be held on August 15th (Sunday).
 
-During the _Bootcamp_ the students and teachers will participate virtually in a series of presentation and exercises, about 10 hours per day, and the students will go through 16 _Small Learning Units_. 
+During the _Bootcamp_ the students and teachers will participate virtually in a series of 
+presentation and exercises, about 10 hours per day, and the students will go through 16 _Small 
+Learning Units_. 
 
-Even though this is a remote edition, we expect everyone to be virtually present during the full length of the bootcamp. Note that these are 2 intensive days where you should be available for the 10 hour period below so make sure to get a good amount of rest!
+Even though this is a remote edition, we expect everyone to be virtually present during the full 
+length of the bootcamp. Note that these are 2 intensive days where you should be available for the 
+10 hour period below so make sure to get a good amount of rest!
 
-**If you miss the Bootcamp or Hackathon #1, you won't be allowed to participate in any other Starters Academy activities.**
+**If you miss the Bootcamp or Hackathon #1, you won't be allowed to participate in any other Starters 
```

Comment:
I think this should probably be told to them in the first contacts, so that they know what they're signing up for. @CatarinaSilva @thecarolmartins  what do you think?

Reply:
Yes, this should be mentioned before, and in the intro session, that's the goal. To avoid confusion, those exceptions should be mentioned at that point, and here I would leave it strict

Reply:
I believe Carol answered below and added FAQs pointing to the wiki in the website. For the intro @rjna should be aware of it